web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -833,6 +1050,20 @@ function Subcription({
             />
           </Card.Body>
         </Card>
+        <BillingEmailForm
+          className="mb-4"
+          defaultValue={subscription.billingEmail}
```

Comment:
We don't update the parent data on save. I think this is okay and it makes things easier to implement.

The slight downside is that if you edit the billing email and open the "Modify Subscription" modal the new email will not be reflected.